packages/sources/intrinio-test/src/endpoint/price.ts

```diff
@@ -0,0 +1,51 @@
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import { EndpointTypes } from './price-router'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+
+const logger = makeLogger('Intrinio Price')
+
+export const batchTransport = new HttpTransport<EndpointTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((symbol) => {
+      const requestConfig = {
+        baseURL: config.API_ENDPOINT,
+        url: `securities/${symbol.base}/prices/realtime`,
+        params: {
+          api_key: config.API_KEY,
+        },
+      }
+      return {
+        params: [symbol],
+        request: requestConfig,
+      }
+    })
+  },
```

Comment:
Is this a batch request, or is this a regular single request? Asking because this is named `batchTransport` on L7

Reply:
It's a single request, I'll call it `httpTransport`, is that okay with you?

Reply:
Yes